previewer/metadata_renderer.js

```diff
@@ -45,6 +51,8 @@ setMetadataAssetID = function(ID) {
     )
 
     visibleProps.forEach(function(prop) {
+      if (!asset.hasOwnProperty(prop))
+        return;
```

Comment:
@ptomato - yeah no idea about that. maybe we alway show the common ones and then if there's anything left over we append it to the list.